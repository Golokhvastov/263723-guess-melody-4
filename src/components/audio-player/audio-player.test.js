import React from "react";
import renderer from "react-test-renderer";
import AudioPlayer from "./audio-player.jsx";

const mock = {
  src: `https://upload.wikimedia.org/wikipedia/commons/4/4e/BWV_543-fugue.ogg`,
  isActive: true,
};

it(`AudioPlayer render correctly`, () => {
  const tree = renderer
    .create(
        <AudioPlayer
          src = {mock.src}
          isActive = {mock.isActive}
        />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
