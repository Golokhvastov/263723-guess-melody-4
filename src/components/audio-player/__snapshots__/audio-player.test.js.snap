// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AudioPlayer render correctly 1`] = `<audio />`;
